go-controller/pkg/clustermanager/userdefinednetwork/userdefinednetwork_controller.go

```diff
@@ -166,6 +181,21 @@ func (c *Controller) SyncUserDefinedNetwork(udn *userdefinednetworkv1.UserDefine
 	return nil
 }
 
+func validatePrimaryNetworkNadNotExist(nads []*netv1.NetworkAttachmentDefinition) error {
```

Comment:
I think this comment should be somewhere in code, so if someone tries to change any of the mentioned parameters, they know the consequence. I picked this function as I hope when we add another cudn controller, it will reuse the same function, and we will stumble on this comment. But you can pick some other place if you find it more appropriate

Reply:
Done